ESS/v7/_init.yaml

```diff
@@ -0,0 +1,303 @@
+instances:
```

Comment:
Thanks for following the procedure for increasing the version number when adding new telemetry. As it happens, v6 is not in use yet at the summit so please use that version instead. Once done please re-request my review.

Reply:
Hi @wvreeven I'm so sorry - I didn't see this comment until now. Are you saying I should put this in v6? Since this comment was made a month ago, is this still the case?

Reply:
v6 has hit the summit now so v7 for your changes is correct.